<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginCamera</class>
 <widget class="QWidget" name="LoginCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginCamera</string>
  </property>
  <widget class="QLabel" name="label_photo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>77</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="push_button_make_another_photo">
      <property name="text">
       <string>&amp;Make photo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_button_accept_photo">
      <property name="text">
       <string>&amp;Accept photo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
